.github/workflows/memory_statistics.yml

```diff
@@ -0,0 +1,22 @@
+name: Memory statistics
```

Comment:
This separate workflow of generating memory statistic values is not required as the PR check ensures that the memory statistics table is always up-to-date.